nservicebus/pipeline/manipulate-with-behaviors_shareddata_core_[5.0,).partial.md

```diff
@@ -1,6 +1,8 @@
 
 ## Sharing data between Behaviors
 
-Sometimes a parent behavior might need to pass some information to a child behavior and vice versa. The `context` parameter of a behavior's Invoke method facilitates passing data between behaviors. The context is very similar to a shared dictionary which allows adding and retrieving information from different behaviors.
+Sometimes a parent behavior might need to pass some information to a child behavior and vice versa. The `context` parameter of a behavior's `Invoke` method facilitates passing data between behaviors. The context is very similar to a shared dictionary which allows adding and retrieving information from different behaviors.
 
-snippet: SharingBehaviorData
\ No newline at end of file
+snippet: SharingBehaviorData
+
+Note: Contexts are not concurrency safe.
```

Comment:
Not religious.

Just for my own benefit, what would be a scenario where you would be hit by this?